themes/default/layouts/docs/single.html

```diff
@@ -1,55 +1,49 @@
 {{ define "main" }}
-    <div class="docs-list-main">
-        <div class="docs-main-nav-toggle-wrapper">
-            <div class="docs-main-nav-wrapper">
-                <div id="docs-main-nav" class="docs-main-nav">
-                    {{ partial "docs/main-nav.html" . }}
-                </div>
-            </div>
-            <div class="docs-nav-toggle">
-                <div class="docs-nav-toggle-icon icon icon-24-24"></div>
+<div class="docs-list-main">
```

Comment:
lot of wacky formatting changes - the only actual change is adding `{{ partial "docs/right-nav-ad.html" }}` in various spots.